internal/blueprint/ext4_output.go

```diff
@@ -5,14 +5,28 @@ import "github.com/osbuild/osbuild-composer/internal/pipeline"
 type ext4Output struct{}
 
 func (t *ext4Output) translate(b *Blueprint) *pipeline.Pipeline {
-	p := getF30Pipeline()
+	packages := [...]string{
+		"policycoreutils",
+		"selinux-policy-targeted",
+		"kernel",
+		"firewalld",
+		"chrony",
+		"langpacks-en",
+	}
+	excludedPackages := [...]string{
+		"dracut-config-rescue",
+	}
+	p := getCustomF30PackageSet(packages[:], excludedPackages[:])
+	addF30LocaleStage(p)
+	addF30GRUB2Stage(p, b.getKernelCustomization())
 	addF30SELinuxStage(p)
+	addF30FixBlsStage(p)
```

Comment:
This should be fine, but I would move (in a follow-up) SELinux last, always do the fix bls stuff right after DNF.

Reply:
Nice catch! Fixed this one and added a commit which also flips them for openstack, vhd, and qcow2.

Reply:
Sorry for this one, I wasn't sure about proper usage of `fix-bls` stage.